README.rst

```diff
@@ -671,7 +746,31 @@ Preprocessed images
 Group-level
 ===========
 
--  ``grouplevel/...``
+-  |  For task-based group-level analyses, HALFpipe outputs the statistical maps for the effect, the variance, the degrees of freedom of the variance, and the t-statistic and z-statistic for each linear model that is specified, much as for the first-level features. These models are fit per session, task, first-level feature, first-level task contrast, and group-level contrast, where these exist.
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_stat-effect_statmap.nii.gz``
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_stat-variance_statmap.nii.gz``
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_stat-dof_statmap.nii.gz``
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_stat-t_statmap.nii.gz``
+   |  ``grouplevel/…/func/task-…feature-…taskcontrast-…/model-…/-…/…_task-…_feature-…_taskcontrast-…_model-…_contrast-…_stat-z_statmap.nii.gz``
+   |
+-  |  For every contrast, the mean and standard deviation of the regressor used in the analysis.
```

Comment:
We should explain why these are images, and not just single values (because the mean varies depending on which participants have data for each voxel)

Reply:
Resolved